src/RedialContext.js

```diff
@@ -17,6 +17,8 @@ function hydrate(renderProps) {
 }
 
 export default class RedialContext extends Component {
+  static displayName = 'RedialContext';
```

Comment:
Why do we need this? In `__DEV__` it'll be inferred automatically, and in `production` we don't want/need the name to not be minimized?